src/components/Checkbox.jsx

```diff
@@ -8,7 +8,7 @@ import PropTypes from 'prop-types'
  * @property {boolean | undefined} [checked]
  * @property {boolean | undefined} [disabled]
  * @property {(checked: boolean) => void} [onChange]
- * @property {string | React.ReactNode | React.ReactNode[]} [label]
+ * @property {string} [label]
```

Comment:
Why. Seems we loose some versatility here.

Reply:
For the reason I described above (input valye can't be ReactNode nor ReactNode[]). I've no idea what the original intention could have been, open to suggestions.

Reply:
Yup, I also do not understand what does it mean for the value to be assigned the label. I think there may be a bug here.

Reply:
@henningmu please?

Reply:
We only use string 👍

Reply:
@henningmu So proptypes are wrong and the correct fix is to limit them to string, right?

Reply:
Correct

Reply:
I agree that this PR should be concerned with what we have now, so not meaning to say we should change this here, but...

Why do we pass the checkbox label as its `value` attribute? To the best of my knowledge, I think that the `value` attribute in a `<input type="checkbox" />` element is usually not needed, and it is only used when you submit a form via the browser submit mechanism. But in a SPA we rarely, if ever, do that.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#Value)

> This is never seen on the client-side, but on the server this is the `value` given to the data submitted with the checkbox's `name`.

Could we stop passing it as the `value`, and make it a `React.ReactNode`?

Reply:
@gnapse How about you open another issue about it? Let's focus on getting this shipped here.